code/game/objects/items/vending_items.dm

```diff
@@ -75,11 +75,13 @@
 	charges = list(37, 4, 4)// of 111 standard, 12 contraband, 10 premium(?)
 	init_charges = list(37, 4, 4)
 
+/*

```

Comment:
You don't need to comment out this refill, just make a comment changing the charges of the vendor itself. the override *should* take care of it as well

Reply:
I get the idea behind it - citadel_modular is loaded after the regular game code, thus overriding anything that needs it. However, I think that's undefined behaviour, so I'd rather not rely on it.